src/types/RootState.ts

```diff
@@ -0,0 +1,15 @@
+import { GithubRepoFormState } from 'app/containers/GithubRepoForm/types';
+import { ThemeState } from 'styles/theme/types';
+// [IMPORT NEW CONTAINERSTATE ABOVE] < Needed for generating containers seamlessly
+
+/* 
+  Because the redux-injectors injects your reducers asynchronously somewhere in your code
+  You have to declare them here manually
```

Comment:
:cry:

Reply:
Can't think of any workaround to solve this. At least generators help you here.